src/main/java/be/syntra/devshop/DevshopBack/models/SearchModelDto.java

```diff
@@ -17,10 +17,13 @@
     private BigDecimal priceHigh;
     private boolean sortAscendingName;
     private boolean sortAscendingPrice;
+    private boolean nameSortActive;
+    private boolean priceSortActive;
     private boolean archivedView;
     private boolean searchResultView;
     private boolean searchFailure;
     private boolean activeFilters;
     private String appliedFiltersHeader;
-
+    private Integer pageNumber;
```

Comment:
change to primitive int?

Reply:
in order to check if no pagination info is received i do a null check and then i can apply default values in the searchServiceImpl, if pagination settings are received from the FE a '0' int is a valid value so if it's 0 the default value doesn't need to be set.